controllers/dockermachine_controller.go

```diff
@@ -123,7 +123,8 @@ func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	if err := r.Client.Update(ctx, dockerMachine); err != nil {
+
+	if err := r.Client.Patch(ctx, dockerMachine, patch); err != nil {
```

Comment:
Odd, there is already a deferred patch happening on line 79, is this update even necessary  here?